src/ui/Pricing/SubscriptionSettings/SubscriptionCard/Card.svelte

```diff
@@ -23,8 +23,8 @@
 </script>
 
 <article class="relative fluid" class:green class:orange class:yellow class:wide={isFullAccess}>
-  <h4 class="caption txt-m c-waterloo mrg-l mrg--b">{label}</h4>
-  <h2 class="h4 txt-m mrg-xs mrg--b">{title}</h2>
+  <h4 class="label txt-m c-waterloo mrg-l mrg--b">{label}</h4>
```

Comment:
Removal of `caption` breaks existing desktop UI. Please, make sure it's consistent with Figma design